build_tools/aws-sdk-code-generator/lib/aws-sdk-code-generator/code_builder.rb

```diff
@@ -140,7 +140,7 @@ def async_client_class
     end
 
     def errors_module
-      Views::ErrorsModule.new(service: @service).render
+      Views::ErrorsModule.new(service: @service, service_name: @service.name).render
```

Comment:
Is this necessary since you are already passing the service object? Somewhere down the line you can just get the service name from the service object.

Reply:
Yeah - that's true actually.  I'll remove those and use the passed service